expat/tests/fuzz/Makefile.am

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Can we give this file a license header like [expat/Makefile.am](https://github.com/libexpat/libexpat/blob/master/expat/Makefile.am)?